File: Projects/codechef/CUTBOARD.java
import java.io.*;
import java.util.*;

class CUTBOARD {
	public static void main (String[] args) throws Exception {
		Scanner stdin = new Scanner(System.in);
		int T = stdin.nextInt();
		while (T-- > 0) {
		final int N = stdin.nextInt(), M = stdin.nextInt();
			System.out.println((N-1)*(M-1));
		}
	}
}

File: Projects/codechef/CUTBOARD.kt
import java.io.*
import java.util.*

internal object CUTBOARD {
    @Throws(Exception::class)
    fun main(args: Array<String>) {
        val stdin = Scanner(System.`in`)
        var T = stdin.nextInt()
        while (T-- > 0) {
            val N = stdin.nextInt()
            val M = stdin.nextInt()
            System.out.println((N - 1) * (M - 1))
        }
    }
}
